.eslintrc.js

```diff
@@ -0,0 +1,16 @@
+module.exports = {
+    root: true,
+    parserOptions: {
+        ecmaVersion: 6,
+        sourceType: 'module'
+    },
+    extends: 'eslint:recommended',
```

Comment:
I think the issue was that jscs was using an older version of the AirBnB preset, so upgrading to the current version was created about 3,000 issues in ember-osf alone. In the long-term, I'd like to set up a separate repo just for our custom style. Since we use it in at least 4 places that I know of, I think it's appropriate.